apps/client/src/hoc/withAutofillData.js

```diff
@@ -15,7 +16,13 @@ const withAutofillData = (Component) =>
     const address = sessionContext[topic.slug]?.address;
 
     // Only autofill data if this topic is an STTR-flow
-    if (topic.sttrFile) {
+    if (topic.flow === Flow.olo) {
```

Comment:
Why did you turn this upside down? I would prefer to check if a flow is STTR, because that's the goal, right? To turn every topic into STTR. (Sorry for nitpicking)

Reply:
Exactly! ;-)
The default is now to use the STTR-flow. The _exception_ is the olo-flow. This way we can easily phase it out.